plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -38,4 +48,38 @@ public void execute(String sql)
             throw new RuntimeException("Failed to execute statement: " + sql, e);
         }
     }
+
+    @Override
+    public void start()
+    {
+        super.start();
+        setUpDatabase();
+    }
+
+    private void setUpDatabase()
+    {
+        execute("CREATE DATABASE " + databaseName);
+
+        if (snapshotIsolationEnabled) {
+            execute(format("ALTER DATABASE %s SET READ_COMMITTED_SNAPSHOT ON", databaseName));
+            execute(format("ALTER DATABASE %s SET ALLOW_SNAPSHOT_ISOLATION ON", databaseName));
+        }
+
+        connectToNamedDatabase = true;
```

Comment:
connectToNamedDatabase -> databaseCreated

but ideally the setup would not use `getJdbcUrl()` at all. maybe a TODO comment